app/Task.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    protected $table = 'tasks';
+    protected $fillable = ['name'];
```

Comment:
Coding rule mình xài cho mảng như này e
Tham khảo thêm: https://github.com/framgia/coding-standards/blob/master/vn/README.md#php
```suggestion
    protected $fillable = [
        'name',
     ];
```

Câu hỏi chung cho mấy bạn. Giải thích cho a thuộc tính `$fillable` này có tác dụng gì?

Reply:
Dạ biến $fillable để ngăn chặn lỗi Mass Assignment ạ. Để em sửa lại cho đúng rule